src/WebJobs.Script.WebHost/Diagnostics/SystemTraceWriter.cs

```diff
@@ -77,7 +77,7 @@ public override void Trace(TraceEvent traceEvent)
                 if (traceEvent.Exception is FunctionInvocationException)
                 {
                     FunctionInvocationException fex = (FunctionInvocationException)traceEvent.Exception;
-                    functionName = fex.MethodName.Replace("Host.Functions.", string.Empty) ?? string.Empty;
+                    functionName = string.IsNullOrEmpty(fex.MethodName) ? string.Empty : fex.MethodName.Replace("Host.Functions.", string.Empty);
```

Comment:
What does this fix?

Reply:
FunctionInvocationException  is sent from the Webjobs SDK.  This is making sure we avoid null ref in case  FunctionInvocationException's method name is not set.

Reply:
Just realized that this will change when you rebase -- `SystemTraceWriter` has become `SystemLogger`. Take a look here: https://github.com/Azure/azure-webjobs-sdk-script/blob/8c771e6a7368b5aad5cdea280ca7e099bd863e39/src/WebJobs.Script.WebHost/Diagnostics/SystemLogger.cs#L85